src/components/CommentList/CommentList.scss

```diff
@@ -1 +1,7 @@
-//
+.CommentList {
```

Comment:
why declared class names with uppercase? Please fix it to lower it is not a good practice: `.commentList`